bundler-audit/entrypoint.sh

```diff
@@ -16,4 +16,4 @@ if [ ! `which bundler-audit` ]; then
 fi
 
 echo "\n# Running bundler-audit..."
-sh -c "bundle audit check --update"
+sh -c "bundle audit check --update ${INPUT_ARGS}"
```

Comment:
I don't believe `$INPUT_ARGS` is right here, or at least I can't find it in documentation. In the other ones, we just references the positional params, like `$1`.

Reply:
it's hard to find but they talk about it in the action metadata syntax page: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions#inputs

> When you specify an input to an action in a workflow file or use a default input value, GitHub creates an environment variable for the input with the name INPUT_<VARIABLE_NAME>. The environment variable created converts input names to uppercase letters and replaces spaces with _ characters.

Reply:
also, if you click the "test build" you'll see that this is def working, otherwise bundler-audit would be failing right now.